content/en/docs/architecture/jira.md

```diff
@@ -22,6 +22,22 @@ To rerun the validations, comment `/jira refresh` on the PR. This is mostly usef
 All Openshift bugs must be made in the `OCPBUGS` project. For PRs referencing issues from other projects, a link will be
 created but the rest of the lifecycle management will not be performed for the issue.
 
+## Premerge Verification
+
+The Jira GitHub integration provides a method to mark a PR as `verified` before the PR is merged. This allows the integration
+to move the bug to the `VERIFIED` state upon merge instead of requiring a QE engineer to test and manually move an issue to the
+`VERIFIED` state after the issue moves into the `ON_QA` state. Premerge verfication is strongly preferred.
+
+To mark a PR as `verified`, comment `/verified by reason` on the PR. The reason can be the names of tests, GitHub usernames,
```

Comment:
Who is permitted to run the command?

Reply:
At the moment, anyone can. What criteria should be used to determine permissions?

Reply:
The integration now only allows collaborators to run the command.